src/domain/authorization/primitives.ts

```diff
@@ -0,0 +1,15 @@
+export const Role = {
+  AccountOwner: "account.owner",
+} as const
+
+export const Permission = {
+  WalletView: "wallet:view",
+  WalletOnChainAddressCreate: "wallet:onChainAddressCreate",
```

Comment:
in my mind I would have use the pattern `onchain:create` or `onchain:send`

Reply:
Should definitely preface it with the resource but good idea for improvement.
Perhaps:
```
wallet:onchain:createAddress
wallet:onchain:send
```